### PR TITLE
Add title to campaign page

### LINKED ISSUE #988: [Bug] Campaign title is not shown in the document title
**Describe the bug**

The campaign title is not shown in the document title which leads to poor SEO and user experience.

**To Reproduce**

1. Go to https://podkrepi.bg/campaigns/staya-za-ergoterapiya-v-pomosh-na-deca-s-narusheno-zrenie-ot-suunz-gr-varna
2. See default page title

**Expected behavior**
See the name of the campaign as title

**Screenshots**
image

### PR DESCRIPTION
**Pull request by kachar**
Closes #988

## Motivation and context

The campaign title is not shown in the document title which leads to poor SEO and user experience.

## Screenshots:

image

## Testing

1. Go to the campaign page of individual campaign
2. Check the title - should match the campaign title

### CODE REVIEW

File: src/components/campaigns/ViewCampaignPage.tsx [hunk 1]
```diff
@@ -49,7 +49,11 @@ export default function ViewCampaignPage({ slug }: Props) {
   const ogImageUrl = campaignListPictureUrl(campaign)
 
   return (
-    <Layout maxWidth={false} ogImage={ogImageUrl} metaDescription={campaign.title}>
+    <Layout
+      maxWidth={false}
+      ogImage={ogImageUrl}
+      title={campaign.title}
```
Comment (igoychev) [inline src/components/campaigns/ViewCampaignPage.tsx:55]: I tried this before, but it has a side effect of the title showing over the campaign image too, so it looks like more changes are needed?

Comment (kachar) [reply src/components/campaigns/ViewCampaignPage.tsx:55]: @igoychev Thanks for mentioning! 

image


Indeed it looks inadequate, will find a solution

Comment (kachar) [reply src/components/campaigns/ViewCampaignPage.tsx]: @igoychev 

I've added the option to pass only `metaTitle` and `title` separately:

```tsx
<Layout
  title="campaign.title"
  metaTitle="meta.title"
/>
```

image

File: src/components/layout/Layout.tsx [hunk 2]
```diff
@@ -36,15 +46,14 @@ export default function Layout({
   figmaUrl,
   hideFooter = false,
   boxProps,
+  metaTitle,
```
Comment (igoychev) [inline src/components/layout/Layout.tsx:47]: it looks like adding metaTitle forces you to use metaTitleInternal and this makes the code confusing. Could we name one of them headTitle to indicate the purpose better?

Comment (kachar) [reply src/components/layout/Layout.tsx:47]: @igoychev I'd rather rename it to `pageTitle` as `headTitle` and `metaTitle` are synonyms and it's also confusing.

**Review by igoychev — APPROVED** [review]

**Pull request merged**